c_common/models/compressors/src/bit_field_compressor.c

```diff
@@ -65,8 +65,8 @@ compressor_states previous_compressor_state = UNUSED_CORE;
 //! SDRAM are used for communication between sorter and THIS compressor
 comms_sdram_t *restrict comms_sdram;
 
-// DEBUG stuff
-#if 0
+// DEBUG stuff please leave
+#ifdef DEBUG_COMPRESSOR
```

Comment:
Is DEBUG_COMPRESSOR defined anywhere?  The only debug flags I can see defined are FEC_DEBUG and PRINT_DEBUG.  (I guess the user can choose to turn this on from their command line "make" but it would probably make sense in that instance to have an action build where this and similar other options are turned on).

Reply:
No, it's not defined. I suggest that these are a group of messages that it makes sense to review for possible deletion as a set. There's three such groups in the extra monitor. Suggesting that _some_ be deleted but others kept is also fine. I don't think that all the messages should be turned on.

Reply:
Ugh; this is the compressor. Same principle though.

Reply:
I'd be inclined to keep all of them but perhaps use the same flag everywhere.  Maybe even the same flag (#if LOG_LEVEL >= LOG_DEBUG) that's used in sPyNNaker, rather than a separate one for each.